pages/general-information-and-resources/tech-policies/tts-pages.md

```diff
@@ -0,0 +1,84 @@
+
+>This is performed after the ATU request is submitted
+
+## 	Reassessment 
+A Site’s ATU will have to be reassessed and an ATU reissued if the Site is found `NOT` to be in conformity with the requirements within this guide.
+
+Conditions/events that may require a reassessment of the ATU include:
+- New third party integrations not on the approved list are added.
+- The data types or information presented on the site changes.
+- A significant security incident occurs.
+- There are deviations from the ATU maintenance requirements.
+
+>This Determination is made by the TTS Pages System Owner `tts-tech-operations@gsa.gov`. Generally, this is done if security findings are not being addressed promptly.
+
+Website Managers will be notified, the following steps are only in the event that the Website Manager is none responsive.
+
+## Failure to Maintain Site - Site Removal
+Sites that fail to maintain the ATU requirements will be issued a formal notice. The TTS Pages team may take steps to disable the site or remediate the vulnerabilities. ATU site owners who hit certain triggers of overdue POA&Ms and/or failure to maintain alignment to ATU requirements will be required to provide a **Corrective Action Plan (CAP)** addressing the plan to address the deficiencies. 
+
+The CAP must be approved by the Site owner, System Owner, ISSM, and IST Director. Sites or Site owners who fail to respond to a CAP, or complete approved actions will be removed from the ATO boundary, and will no longer be authorized. The removal process steps are further described below: 
+- **Detailed Finding Review (DFR)** - Site owners will be issued a DFR upon failing to address a deficiency within the site or alignment with the ATU requirements.
+- **Corrective Action Plan** - Site Owners who fail to adequately respond or address a DFR, will be issued a CAP request. 
+The Site Owner must provide a CAP to the System owner within 30 days of the CAP request. The CAP must detail how the team will address the deficiencies and the timeline for completion.
+
+The Site Owners CAP must be approved by the TTS Pages system owner, the ISSM, and IST Director.
+
+## Site Disablement
+Site Owners who fail to respond to the CAP within the 30 day timeframe, or fail to provide an adequate CAP, or fail to comply with the provisions, timeline and duration of their CAP will have their site Disabled.
+- Disabling a site consists of removing the site within the Cloud.gov Pages Platform which will result in a site being unreachable. 
+
+## Site Removal
+Site Owners who fail to address deficiencies within 90 days of disablement will have their site removed from the TTS Pages ATO boundary and the site will be deleted. 
+- Deleting a site removes the published site from TTS Pages servers and from the dashboards of all site users. This will bring the entire site offline and make it inaccessible for users.
```

Comment:
```suggestion
- Deleting a site removes the published site from the Cloud.gov Pages platform and from the dashboards of all site users. This will bring the entire site offline and make it inaccessible for users.
```

Reply:
accepted